tests/test_indexer.py

```diff
@@ -675,22 +675,39 @@ def test_scrub_index_data(self):
             )
 
         with self.subTest("with invalid schema_url"):
+            url = "http://invalid_url"
```

Comment:
This is actually a valid URL. If there were a host named "invalid_url" in the local network and that host would be running an HTTP server, this URL would work.

Try `"://invalid_url"`–valid URL must have a non-valid scheme and I don't think this would pass for a relative URL either.

Reply:
I *might* be wrong about `invalid_url` being a valid hostname but to be safe if `"://invalid_url"`produces an exception I would go with that.